packages/cursorless-engine/src/languages/LanguageDefinition.ts

```diff
@@ -141,3 +151,15 @@ function readQueryFileAndImports(languageQueryPath: string) {
 
   return Object.values(rawQueryStrings).join("\n");
 }
+
+/**
+ * Returns true if the given scope type should use a contiguous scope handler.
+ */
+function useContiguousScopeHandler(scopeType: ScopeType): boolean {
+  switch (scopeType.type) {
+    case "comment":
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
I thought you were using a query predicate for this now

Reply:
We use both. I don't know how to check it at this stage. When we actually fetch the matched scope it's to late to inject this handler. Not without a lot of rewrites at least.

Reply:
Sorry I'm not following how you would use both. When do you use one vs the other?

Reply:
This code decides to use the handler. The the scope.contiguous is used to determine when to merge two scopes. That way we can ignore block comments.
https://github.com/cursorless-dev/cursorless/blob/b6ac60f708834daa151b24cf4d3f86f7728c4c8c/packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/ContiguousScopeHandler.ts#L119